Design & translation improvements

LehanMusthafa opened the pull request:
A new and fresh look to the website with new colours and fonts.
Here are some key highlights of the new and fresh look:

- New colours and fonts - Syne and Inter(It's cool fonts😎)
- Fresh look to the navigation bar with a download button. So even on a 404-page people can download Brackets.
- New look at the banner and moved it to the bottom with a dismiss button.
- Add a temporary link to the download button which redirects to: https://github.com/adobe/brackets/releases/tag/release-1.14.2 (as mentioned in issue: [couldn't download brackets from brackets.io #17](https://github.com/brackets-cont/brackets/issues/17) & [Can't download #19](https://github.com/brackets-cont/brackets/issues/19) )
- Added a features section with the latest design
- New button design which matches the primary colour.
- Improvements in translations in languages like Italian, Japanese, etc.
- Reformated code
- And many many more improvements.

### A quick preview of the redesigned website
![Frame 11](https://user-images.githubusercontent.com/73033947/133881897-d1b88723-1d13-4491-b222-0d9c637714ed.png)
![features-section](https://user-images.githubusercontent.com/73033947/133881996-a0e014a0-7512-4005-a7fa-f41aaac844b3.png)

yogo1212 commented on locales/de/translation.json:
```diff
@@ -20,25 +20,26 @@
                 "why": {
                     "header": "Warum Brackets?",
-                    "content": "Brackets ist ein einfacher, aber dennoch leistungsfähiger Editor. Er blendet dir hilfreiche Tools immer dann ein, wenn sie gerade passen, ohne dich jedoch von deiner kreativen Arbeit abzulenken. Du wirst es lieben, Code mit Brackets zu schreiben."
+                    "content": "Klammern sind ein leichter, doch leistungsstarker, moderner Texteditor.Wir mischen visuelle Werkzeuge in den Editor, sodass Sie die richtige Menge an Hilfe erhalten, wenn Sie es wünschen, ohne Ihren kreativen Prozess in den Weg zu bringen.Sie werden gerne Code in Klammern schreiben."
```
I don't thing the name should be translated :grin:

ScanuNicco replied:
I don't know much about translations but I agree that the name should not be translated.

yogo1212 commented on locales/de/translation.json:
```diff
@@ -111,10 +112,10 @@
         }
     },
     "nav": {
-        "contribute": "Mach mit",
```
"beitragen" is definitely a very unusual word for this kind of thing, "mitmachen" is better (i dont like imperative "mach mit")

yogo1212 commented on locales/ja/translation.json:
```diff
@@ -28,17 +28,18 @@
                     "inline-editor": {
                         "header": "インラインエディター",
-                        "content": "Brackets では対象となるコードの中にウィンドウが開くため、ファイルタブ間を切り替える必要がありません。特定の ID に適用される CSS で作業する場合その ID にマウスカーソルを合わせ、<kbd>Command<\/kbd> \/ <kbd> Ctrl + E <\/kbd> キーを押すと、その ID を持つすべての CSS セレクターがインラインウィンドウに表示されます。そのため、ポップアップではなく、コードは左右に並べて作業できます。"
+                        "content": "ファイルタブ間でジャンプする代わりに、ブラケットを使用すると、最も気にかけているコードにウィンドウを開くことができます。"
```
i don't know anything about japanese - but the name "Brackets" vanished here as well. is that intentional?

luke2m replied:
"ブラケット" is japanese for "bracket". I'm not sure, but I think it might be best to keep the name in English. I'm pretty busy right now but will look over the translations more soon.

yogo1212 commented on 404.html:
```diff
@@ -106,7 +106,7 @@
             </a>
           </nav>
           <div class="navigation-link-right desktop-link">
-            <a href="https://www.github.com/brackets-cont/brackets">
```
i'm not sure this should be merged - time is likely better spent on creating a release in the brackets-cont repo.

yogo1212 commented:
Hi :-)

Interesting!
There hasn't been an agreement on how we want things to work/look like yet.

I don't know enough about the "inherited" values to be able to tell whether this fits.

@luke2m @ScanuNicco What do you think?
Do you have some to take a look at this?

luke2m commented:
I'm on mobile and busy now so just looking at the screenshots, not the code, but the design looks pretty good to me. I'll take a look at the japanese translation and website code later. I'd appreciate if someone else who knows the languages check those out. Thanks @LehanMusthafa , it seems like you put a lot of work into this!

ScanuNicco commented:
Thanks for the PR @LehanMusthafa, I really like this new design. 

I looked through the code and it mostly looks good to me, but I  think that there needs to be a shadow or something to show where the nav-menu ends on mobile, because it can be kind of hard to distinguish right now:  
![image](https://user-images.githubusercontent.com/30136040/133905947-aaf755a7-53e9-4933-ba80-dc4c18a3c26a.png)

This is with `box-shadow: 0px 3px 6px rgba(0, 0, 0, .1);` added on line 346 of brackets.io.css:
![image](https://user-images.githubusercontent.com/30136040/133906223-d1354664-5a9c-45fd-a1bf-bc44edbcb02d.png)

Also the existng download button is broken because the new S3 bucket doesn't support HTTPS yet, so a better solution for the download button is to temporarily disable HTTPS on brackets.io. It's less secure but the site doesn't have accounts or any other user-info so it's not a huge deal.

LehanMusthafa commented:
Thanks for all your support @yogo1212 @luke2m @ScanuNicco. Let me look into those translation issues and the issues of navbar and download button and try to solve it. I will soon create a commit after solving all these issues.

Thanks again🙂

LehanMusthafa commented:
Hey there everyone!

I have addressed all of your suggestions and changed the code according to that. These are the changes made:

- Reverted changes to the download button link
- Changed the translations in _**de**_ and _**ja**_
- Added shadow to `.show-menu`

ScanuNicco reviewed (approved):
Looks good to me

luke2m commented on index.html:
```diff
@@ -75,148 +101,318 @@
+              <a
+                id="other-downloads"
+                class="nobr"
+                href="https://github.com/brackets-cont/brackets/releases"
```
```suggestion
                href="https://github.com/adobe/brackets/releases"
```
We don’t have binaries or anything past adobe’s releases on our repo.

luke2m reviewed (commented):
Looks good! Just one link fix — I’ll edit and merge.

luke2m commented:
Thanks @LehanMusthafa
